### ISSUE TITLE
Refresh all when server starts

### ISSUE THREAD

**Issue by aknoerig** [napcs/node-livereload#79]
## What version are you using? (Don't say latest. Be specific.)
0.5.0

## Expected result
Refresh all browser views when the node server restarts.

## Actual result
The livereload socket reconnects, but it doesn't refresh the page.

## Why is this important?

Livereload works fine when I edit client-side files. However it would be just as useful when I have to edit and restart the node server. This use case happens almost as often. In my case, I have to reload all open browser views manually in order to re-initialize them (for example to establish further websocket connections).

Basically, I'm looking for a command to force the refresh of all browser views, so that I can call it on server start (and possibly on other occasions).

**Comment by napcs** [comment]
So here's the issue with this.

When you refresh the page manually, or when LiveReload refreshes it, it's a full refresh:

```
Refresh: /Users/brianhogan/dev/oss/node-livereload/.git/index

Browser connected.

Browser URL: {"command":"hello","protocols":["http://livereload.com/protocols/official-6","http://livereload.com/protocols/official-7"],"ver":"2.0.8","snipver":"1"}

```

So, whenever the HTML page refreshes, it literally makes a new opening connection to the server.

If we tell the LiveReload server to refresh your browser when it connects, then it will refresh infinitely because once it refreshes, it'll make the socket connection, triggering reloading. 

Run LiveReload server separate from your Node application so you don't have to stop it. This works for me.

**Comment by aknoerig** [comment]
Thanks, I have now opted for an alternative solution. Since my issue is mostly about re-establishing other websocket connections (apart from the one livereload uses), I'm using this code:

http://stackoverflow.com/questions/3780511/reconnection-of-client-when-server-reboots-in-websocket
